app/frontend/resources/assets/sass/app.scss

```diff
@@ -13,6 +13,9 @@
 @import '~bootstrap4c-chosen/src/scss/component-variables';
 @import '~bootstrap4c-chosen/src/scss/component-chosen';
 
+// simple-lightbox
+@import '~simple-lightbox/dist/simpleLightbox.min';
```

Comment:
is this specifically css? you may need to suffix with `.css` explicitly

Reply:
Also as a rule of thumb you shouldn't use `dist` packages in this bit, see if there's a `src` or scss version in case we want to customize it.

Reply:
SASS looks for .css as well as .scss - I've confirmed it loads locally. There *is* a scss version in src but it requires another node module (`bourbon`) to build and that seemed a little silly for a pretty lightweight stylesheet.